reference-architecture/osp-cli/docker-storage-setup

```diff
@@ -0,0 +1,2 @@
+DEVS=/dev/vdb
+VG=docker-vg
```

Comment:
I agree, but to use cloud-init well in the refarch Id have had to also pull in core-os/cloud-init/tools/make-mime.py.  I can't find a RHEL package with make-mime.py and I didn't want to make the cloud-init script a gigantic mess with two or three replacements.

Several other things can go in cloud-init if we can dynamically create the host-data as a multi-part MIME file in a simple way.